### PR TITLE
First two chapters are tested

### PR DESCRIPTION
**Pull request by sarveshpandit**
First two chapters are tested and suggested changes.

### CODE REVIEW

File: modules/ROOT/pages/index.adoc
Comment (rsriniva) [inline modules/ROOT/pages/index.adoc:1]: good catch. merge these changes!

File: modules/chapter1/pages/section3.adoc [hunk 1]
```diff
@@ -3,17 +3,18 @@
 
 In this exercise, you will use RHOAI to train and evaluate a simple machine learning model, by using JupyterLab and other common ML libraries.
 
+// Not understood the demo. Can be moved in next chapter before Lab.
```
Comment (rsriniva) [inline modules/chapter1/pages/section3.adoc:6]: OK. Good suggestion. Please cut the demo from here and move it to the appropriate place before final Lab

File: modules/chapter1/pages/section3.adoc [hunk 2]
```diff
@@ -93,6 +102,7 @@ Keep pressing btn:[Shift+Enter] until you reach the bottom.
 5. *Run the `intro-text-generation` notebook*
 
 a. In the data science project dashboard, create a workbench with the default image for PyTorch.
+// Need to add details such as name other settings.
```
Comment (rsriniva) [inline modules/chapter1/pages/section3.adoc:105]: If you think something is missing, please add details

File: modules/chapter2/pages/section3.adoc [hunk 3]
```diff
@@ -104,6 +104,7 @@ Model Serving downloads the model files from S3 by using a data connection.
 Before you start with this exercise, you must configure an S3 bucket.
 
 If you do not have an S3 object store available, follow the https://ai-on-openshift.io/tools-and-applications/minio/minio/[guide to create an S3 bucket by using _Minio_].
+// Can we add separate page on deploy Minio?
```
Comment (rsriniva) [inline modules/chapter2/pages/section3.adoc:107]: no need. if the linked doc is good enough, then no need to duplicate

File: modules/chapter2/pages/section3.adoc [hunk 4]
```diff
@@ -190,3 +194,4 @@ c. Follow the instructions in the notebook.
 Click the first cell, then press btn:[Shift+Enter] to execute the cell and move to the next one.
 Next, execute and review the rest of the cells.
 Keep pressing btn:[Shift+Enter] until you reach the bottom.
+// Exercise has not end objective or goal that we have achieved.
```
Comment (rsriniva) [inline modules/chapter2/pages/section3.adoc:197]: Please add appropriate objective to beginning and summarize at the end of the exercise

File: modules/chapter2/pages/section4.adoc [hunk 5]
```diff
@@ -2,6 +2,7 @@
 :navtitle: Lab
 
 In this exercise, you will practice the use of data science projects, permissions, workbenches, environment variables, and data connections.
+//Need more information on the objectives and goals of the lab.
```
Comment (rsriniva) [inline modules/chapter2/pages/section4.adoc:5]: hmm - maybe the objective is to learn these basic concepts of projects, permissions, workbenches, env vars and connections? please suggest alternative objectives

File: modules/chapter2/pages/section4.adoc [hunk 6]
```diff
@@ -55,6 +56,8 @@ https://raw.githubusercontent.com/RedHatQuickCourses/rhods-qc-apps/main/1.intro/
 == Solution
 
 1. Create the projects.
+// Login as user1 is missing in the instruction.
```
Comment (rsriniva) [inline modules/chapter2/pages/section4.adoc:59]: please add instructions for log in

File: modules/chapter2/pages/section4.adoc [hunk 7]
```diff
@@ -55,6 +56,8 @@ https://raw.githubusercontent.com/RedHatQuickCourses/rhods-qc-apps/main/1.intro/
...
+// When to run the command is missing -  "$ oc create -f https://raw.githubusercontent.com/RedHatQuickCourses/rhods-qc-apps/main/1.intro/chapter2/lab/group.yaml"
```
Comment (rsriniva) [inline modules/chapter2/pages/section4.adoc:60]: please add it

**Review by rsriniva — CHANGES_REQUESTED** [review]
excellent work Sarvesh! please review my comments and apply the changes. when fully done, ping me again for a quick approval. we can then merge and ask Rutuja to start

**Comment by rsriniva** [comment]
Other adoc files with no review comments - please merge the suggestions because I think they are good

File: modules/ROOT/pages/index.adoc [hunk 8]
```diff
@@ -13,6 +13,7 @@ This course is the *first* in a series of *five* courses about Red Hat OpenShift
 NOTE: After you have completed all the courses in the learning path, you can attempt the https://github.com/RedHatQuickCourses/rhods-qc-apps/tree/main/7.hands-on-lab["Workshop final hands-on lab"], that tests your understanding of the concepts taught in all the courses.
 
 IMPORTANT: The hands-on labs in this course were created and tested with RHOAI v2.4. Labs should mostly work without any changes in minor dot release upgrades of the product. Please open issues in this repository if you run into any issues.
+// I see Operator version as 2.8.0, can we change from 2.4 to 2.8.0?
```
Comment (rsriniva) [inline modules/ROOT/pages/index.adoc:16]: yes. please update

File: modules/chapter2/pages/section4.adoc [hunk 9]
```diff
@@ -1,151 +1,110 @@
...
 
-d. Log out and log back in with both `user4` and `user5` users.
-Verify that `user4` can access the `intro-projects-b` project.
-Next, verify that `user5` cannot access the `intro-projects-b` project.
+b. In the new workbench, open JupyterLab and clone the https://github.com/RedHatQuickCourses/rhods-qc-apps.git repository as you did in the previous step.
+This demo shows you how to fine-tune a pretrained language model for text generation by using _Causal language modeling_.
 
+c. Open the `rhods-qc-apps/1.intro/chapter1/intro-text-generation/notebook.ipynb` notebook in JupyterLab and execute the steps.
+The instructions are embedded in the notebook.
+// This is not working as expected. I get this error - "ls: cannot access 'my_model/pytorch_model.bin': No such file or directory"
```
Comment (rsriniva) [inline modules/chapter2/pages/section4.adoc:110]: not sure. none of the other users have complained. please open an issue

**Review by rsriniva — APPROVED** [review]
Approved. please merge after addressing my comments

File: modules/ROOT/pages/index.adoc (hunk 8 above)
Comment (sarveshpandit) [reply modules/ROOT/pages/index.adoc:16]: Okay.

**Pull request merged**